Serializer/Normalizer/ExceptionHandler.php

```diff
@@ -102,7 +102,10 @@ protected function convertToArray(\Exception $exception, Context $context)
 
         $templateData = $context->attributes->get('template_data');
         if ($templateData->isDefined()) {
-            $data['code'] = $statusCode = $templateData->get()['status_code'];
+            $data = $templateData->get();
```

Comment:
ah, i guess thats also the reason for the failing tests... was not looking close enough to see that $data is already used here.